src/Backend/EdgeTPU/EdgeTPUToolchain.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command } from "../Command";
+import {
+  DebianToolchain,
+} from "../ToolchainImpl/DebianToolchain";
+
+import * as ini from "ini";
+import * as fs from "fs";
+
+class EdgeTPUDebianToolchain extends DebianToolchain {
+  run(cfg: string): Command {
+
+    let cmd = new Command("edgetpu_compiler");
+
+    // TODO: cfg는 cfg file path임
+    //       cfg 파일을 열어 edgetpu_compiler 명령 옵션 이어 붙이기로 Command를 만들어 반환해야함
```

Comment:
It seems that this TODO has been implemented in the current commit, so it can probably be removed.

Reply:
I forgot to delete them. Thanks.